### DISCUSSION TITLE
Decode CLUSTD

### DISCUSSION THREAD

**Discussion by tbanerjee2** [SAP/PyRFC#272] [Q&A] [answered]
Hi,

is there a feature to decode the CLUSTD column of SAP, typically we use read_text method. 

Thanks

**Comment by tbanerjee2** [comment]
Hi,

Can anyone please answer this question, sorry I may have less knowledge on ABAP.

**Reply by bsrdjan** [reply]
Hi @tbanerjee2,

the question is about ABAP and PyRFC has no special methods for CLUSTD. You can check if it is already answered in SAP ABAP community and post the question there:

https://community.sap.com/search/?ct=qa&q=Clustd

**Reply by tbanerjee2** [reply]
Hi @bsrdjan thanks, but would like to know if I can use the Function module READ_TEXT in Pyrfc?

**Comment by bsrdjan** [comment, marked as answer]
The `READ_TEXT` function module is not remote enabled (flag not set) and can't be called via RFC protocol, neither from ABAP, nor from Python. ABAP developer can provide a RFC enabled wrapper, that can be called via RFC and internally calling `READ_TEXT`.
